### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by jaredmorrow** [akoutmos/prom_ex#84] [General]
There have been some [very useful features added since 1.3.0](https://github.com/akoutmos/prom_ex/compare/1.3.0...master), is it possible to get a release (1.3.1?) made with those changes?  

Thanks so much, and thanks for all the work on prom_ex!

**Comment by akoutmos** [comment]
Hey @jaredmorrow! It's on my todo list for the next week or so. Given how many things were added to PromEx recently, this release will require a fair amount of testing which I have not yet gotten to. I also need to determine whether I want this to be a 2.0.0 release since the Phoenix plugin now has some breaking changes. If you have some bandwidth and want to test all the latest code out that would be much appreciated!

**Reply by jaredmorrow** [reply]
Thanks, I totally understand.  I'll move one of our projects to head and test out some of the newer changes by hand.

**Reply by akoutmos** [reply]
That would be super helpful! Thanks a ton!

**Reply by jaredmorrow** [reply]
Tested and adjusted some things in #85

**Reply by jaredmorrow** [reply]
Otherwise everything works as expected

**Reply by akoutmos** [reply]
Awesome! Thanks for taking it for a test drive :)

**Comment by sarasfox** [comment]
Is this project still active and/or can I help?
